src/main/resources/chess_board.fxml

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
+<?import javafx.scene.text.Text?>
+<?import java.util.ArrayList?>
+<GridPane xmlns="http://javafx.com/javafx"
+          xmlns:fx="http://javafx.com/fxml"
+          fx:id="chessBoardView"
+          stylesheets="@chess.css"
+          maxHeight="-Infinity" maxWidth="-Infinity"
+          minHeight="-Infinity" minWidth="-Infinity"
+          prefHeight="800.0" prefWidth="800.0">
+
+    <ToolBar GridPane.rowIndex="0" GridPane.columnSpan="12" GridPane.columnIndex="0">
```

Comment:
Do not to more than was asked. You were asked to create an xml for the Board, not a menu.

Reply:
Removed extra code in latest commit